### DISCUSSION TITLE
Connection attempt timed out. Connection attempt timed out.

### DISCUSSION THREAD

**Discussion by CCCCCCCT** [dbeaver/dbeaver#22462] [DBeaver Development]
![image](https://github.com/dbeaver/dbeaver/assets/145830515/0078294d-b2e8-4cc8-aa0d-c800094f00ce)

I want to connect to my VMWare ubuntu22.04 through dbeaver. I loaded docker in ubuntu, created postgreSQL in it and tried to connect to the database inside, but it kept showing Connection attempt timed out.
I also tried portqry, and it showed me FILTERED.
The firewall in my ubuntu is not even turned on, and the firewall in win10 does not show any problems.
Can anyone please help me? I would be very grateful.

**Comment by LonwoLonwo** [comment]
Hello @CCCCCCCT 

Maybe this is really connection time out.
Then, you can increase the connection timeout property in your connection settings.

![2024-01-16 10_43_04-DBeaver Ultimate 23 3 3 - _postgres 6 1_ Script-122](https://github.com/dbeaver/dbeaver/assets/45152336/b43da09d-f8e2-4ea4-8152-56aba38a3e12)
